samples/testing/testify/require_test.go

```diff
@@ -0,0 +1,57 @@
+package testify
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRequire(t *testing.T) {
+	// require.New creates an object that can be used lik more common test frameworks
```

Comment:
s/lik/like

explain difference between require and assert:
> Package require implements the same assertions as the `assert` package but stops test execution when a test fails.

I've stopped using this - I use `assert` everywhere and `go test -failfast ./...` if i want tests to bail on first failure....